### PR TITLE
Add the tutorial to submit an AllReduce job on minikube.

### PR DESCRIPTION
**Pull request by workingloong**

### CODE REVIEW

File: docs/tutorials/elasticdl_local.md [hunk 1]
```diff
@@ -99,22 +99,51 @@ retrieve the source code, please run the following command.
...
+
+Then, we can build the AllReduce training image `elasticdl:mnist_allreduce`
+with model definitions in `model_zoo`.
 
 ```bash
-cd elasticdl/model_zoo
-elasticdl zoo init
-elasticdl zoo build --image=elasticdl:mnist .
+elasticdl zoo init \
+  --base_image=elasticdl:dev_allreduce \
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_local.md:136]: Can we ship a base image for AllReduce job at this time? This tutorial is to make users run the job as conveniently as possible.
After horovod releases its pip package, we can use the way of `pip install elasticdl` instead.

File: docs/tutorials/elasticdl_local.md [hunk 2]
```diff
@@ -187,3 +214,50 @@ The output looks like the following.
...
+elasticdl-test-mnist-allreduce-worker-1   1/1     Running   0          98s
+```
+
+Then, we can view the loss in the worker log using
+
+```bash
+kubectl logs elasticdl-test-mnist-allreduce-worker-0 | grep Loss
+```
+
+The output looks like the following.
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_local.md]: The outputs look like

File: docs/tutorials/elasticdl_local.md [hunk 3]
```diff
@@ -187,3 +214,50 @@ The output looks like the following.
...
+related containers.
+
+```bash
+NAME                                      READY   STATUS    RESTARTS   AGE
+elasticdl-test-mnist-allreduce-master     1/1     Running   0          102s
+elasticdl-test-mnist-allreduce-worker-0   1/1     Running   0          98s
+elasticdl-test-mnist-allreduce-worker-1   1/1     Running   0          98s
+```
+
+Then, we can view the loss in the worker log using
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_local.md]: using the following command

File: docs/tutorials/elasticdl_local.md (hunk 1 above)
Comment (workingloong) [reply docs/tutorials/elasticdl_local.md:136]: After AllReduce is stable, we can ship a base image. After Horovod releases its latest package, we can add it to the requirement of ElasticDL.

File: docs/tutorials/elasticdl_local.md [hunk 4]
```diff
@@ -99,22 +99,51 @@ retrieve the source code, please run the following command.
 git clone https://github.com/sql-machine-learning/elasticdl
 ```
 
-The model definitions are in directory `elasticdl/model_zoo`.  The following
-commands build the Docker image `elasticdl:mnist`
+The model definitions are in directory `elasticdl/model_zoo`.
```
Comment (skydoorkai) [inline docs/tutorials/elasticdl_local.md]: The model definitions -> Model definitions

File: docs/tutorials/elasticdl_local.md [hunk 5]
```diff
@@ -99,22 +99,51 @@ retrieve the source code, please run the following command.
...
+
+### Build the Model Zoo for Parameter Server
```
Comment (skydoorkai) [inline docs/tutorials/elasticdl_local.md]: Build the Model Zoo? 
We are building docker image for the job to run.

File: docs/tutorials/elasticdl_local.md (hunk 1 above)
Comment (brightcoder01) [reply docs/tutorials/elasticdl_local.md:136]: We can upload a runnable image at this time. After it's stable, we can overwrite the image with the stable version.

File: docs/tutorials/elasticdl_local.md [hunk 6]
```diff
@@ -99,22 +99,51 @@ retrieve the source code, please run the following command.
...
+The following commands build the Docker image `elasticdl:mnist_ps`
+
+```bash
+cd elasticdl
+elasticdl zoo init --model_zoo=model_zoo
+elasticdl zoo build --image=elasticdl:mnist_ps .
+```
+
+### Build the Model Zoo for AllReduce
+
```
Comment (skydoorkai) [inline docs/tutorials/elasticdl_local.md:115]: We have not released ElasticDL packages with AllReduce yet. Thus, we need to manually build packages with AllReduce support.

We must build an image `elasticdl:dev_allreduce` firstly using the

```bash
scripts/travis/build_images.sh
```

Then we use this image to build packages with AllReduce support.

```bash
scripts/docker_build_wheel.sh
```

After this, we can build the AllReduce training image `elasticdl:mnist_allreduce`
with model definitions in `model_zoo`.

File: docs/tutorials/elasticdl_local.md [hunk 7]
```diff
@@ -99,22 +99,51 @@ retrieve the source code, please run the following command.
...
+```bash
+cd elasticdl
+elasticdl zoo init --model_zoo=model_zoo
+elasticdl zoo build --image=elasticdl:mnist_ps .
+```
+
+### Build the Docker Image for AllReduce
+
+We must build the base image `elasticdl:dev_allreduce` firstly using the
```
Comment (skydoorkai) [inline docs/tutorials/elasticdl_local.md]: firstly -> first

**Review by skydoorkai — APPROVED** [review]

File: docs/tutorials/elasticdl_local.md (hunk 1 above)
Comment (workingloong) [reply docs/tutorials/elasticdl_local.md:136]: It is slow to upload the image. I will use the AllReduce base image in the next PR after I succeed in shipping the image.

**Pull request merged**